src/__tests__/__helpers__/renderWithProviders.tsx

```diff
@@ -58,6 +60,7 @@ const providersByName: {
 } = {
   cookieConsent: [MockedCookieConsentProvider],
   theme: [ThemeProvider, { theme }],
+  oakTheme: [OakThemeProvider, { theme: oakDefaultTheme }],
```

Comment:
This seems reasonable. It did occur to me that others may have used renderWithProviders() and an OakThemeProvider, meaning there could be 2 OakThemeProviders providers now. But I don't think that would cause any issues anyway.

Reply:
Yeah it took me a while to work out why the tests weren't working 🙂